### ISSUE TITLE
Cron node places 6 star placeholders in custom mode instead of 5

### ISSUE THREAD

**Issue by pablorocka** [n8n-io/n8n#1391]
**Cron node places 6 star placeholders in custom mode instead of 5**
In Cron node, selecting Custom mode, places a default of 6 starts (**\***), however, crontab standard only defines 5 (minute, hour, day of month, month, day of week).

**Steps To Reproduce**
1. Add a Cron Node
2. Select "custom" Mode
3. Cron Expression is pre-poulated with 6 stars

**Expected behavior**
When selecting "custom" mode, Cron Expression should be populated with 5 stars

**Environment:**
 - OS: Ubuntu 20.04.1 LTS (GNU/Linux 5.4.0-1028-raspi aarch64)
 - Docker Image: n8nio/n8n:latest-rpi (sha256:a0c61a8f008f0c5bb798dc9f313f8e0e6dacf57db575cf300bc72e100342a4b6)
 - n8n Version  0.104.2

**Comment by janober** [comment]
Thanks a lot. But that is correct and there on purpose. The 6th star is for seconds. It however also works with 5. So you can use 5 or 6 whatever you prefer.

**Comment by pablorocka** [comment]
Thanks, the reason why I noticed this is because I got an unexpected scheduled. I didn't pay attention to the placeholders and simply edited the first and second to set at 0 minutes on 10am, 2pm and 8pm like this:

`0 10,14,20 * * * *`

However, my workflow it was executed at minutes 10, 14 and 20 in the following hour. I can try to reproduce this, but my first thought was that the extra star caused the issue.

i'll test and will report back.
